mobileversion.html

```diff
@@ -0,0 +1,186 @@
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+       <div class="container">
+         <div class="item1">
+         <img src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl.png" alt="snapshoot">
+         <section class="ton">
+            <h1 class="tonic">Tonic</h1>
+         </section>
+         <section class="sec">
+            <p class="mainp">
+            <h5 class="canopy">CANOPY</h5>
+            <p class="mid">&#8226;</p>
+            <p class="back">Back End Dev</p>
+            <p class="mid">&#8226;</p>
+            <p class="year">2015</p>
+            </p>
+         </section>
+         <section class="day">
+            <p class="daily">
+               A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+         </section>
+         <section class="maincon">
+            <ul>
+         <li class="container2"><a href="#">html</a></li>
+         <li class="container3"><a href="#">css</a></li>
+         <li class="container4"><a href="#">javaScript</a></li>
+         </ul>
+      </section>
+      <section class="see">
+         <a href="#" class="pro">See Project</a>
+      </section>
+      </div>
+      </div>
+      <div class="container">
+        <div class="item1">
+            <img src="C:\Users\Harrison\Downloads\Snapshoot Portfolio (3).png" alt="multiimage2">
+        <section class="ton">
+            <h2 class="mps">Multi-Post<br>
+                Stories</h2>
+        </section>
+        <section class="sec">
+           <p class="mainp">
+           <h5 class="canopy">CANOPY</h5>
+           <p class="mid">&#8226;</p>
+           <p class="back">Back End Dev</p>
+           <p class="mid">&#8226;</p>
+           <p class="year">2015</p>
+           </p>
+        </section>
+        <section class="day">
+           <p class="daily">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </section>
+        <section class="maincon">
+           <ul>
+        <li class="container2"><a href="#">html</a></li>
+        <li class="container3"><a href="#">css</a></li>
+        <li class="container4"><a href="#">javaScript</a></li>
+        </ul>
+     </section>
+     <section class="see">
+        <a href="#" class="pro">See Project</a>
+     </section>
+     </div>
+     </div>
+     <div class="container">
+          <div class="item1">
+            <img src="https://mir-s3-cdn-cf.behance.net/project_modules/1400/76d179100768775.5f0ff53c7c069.png" alt="gym">
+          <section class="ton">
+             <h1 class="tonic">Tonic</h1>
+          </section>
+          <section class="sec">
+             <p class="mainp">
+             <h5 class="canopy">CANOPY</h5>
+             <p class="mid">&#8226;</p>
+             <p class="back">Back End Dev</p>
+             <p class="mid">&#8226;</p>
+             <p class="year">2015</p>
+             </p>
```

Comment:
- [ OPTIONAL] **You can use here ul tag with li tag rather than a section tag**
```
<ul class="sec">
            <li class="canopy">CANOPY</li>
            <li class="mid">&#8226;</li>
            <li class="back">Back End Dev</li>
            <li class="mid">&#8226;</li>
            <li class="year">2015</li>
</ul>
```